.github/workflows/build-and-publish.yml

```diff
@@ -59,3 +59,23 @@ jobs:
         github_actor: ${{ secrets.LNB_ACTOR }}
         github_token: ${{ secrets.LNB_TOKEN }}
 
+  test-build:
+    name: "Test published image"
+    needs: build
+    runs-on: ubuntu-22.04
+    if: github.repository == 'hpcc-systems/HPCC-Platform'
+    env:
+      WF_YAML_FILE: az-bvt.yml
+    steps:
+      - name: "Trigger external azure test code"
+        run: |
+          community_tag=$( echo ${{ github.ref }} | cut -d'/' -f3 )
+          hpcc_version=$( echo $community_tag | sed 's/community_//' | sed 's/-[0-9]$//' )
+          curl -L \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ secrets.LNB_TOKEN }}" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/hpcc-systems/github-action-hpcc-terraform/actions/workflows/$WF_YAML_FILE/dispatches \
```

Comment:
assumes we've moved the code over to hpcc-systems/github-action-hpcc-terraform

Reply:
I don't if we still name it github-action-hpcc-terraform or not

Also I don't find "-d" for curl command. It will give github-action-hpcc-terraform repo branch and input, i.e. hpccVersion and hpccSrcBranch

Reply:
I think for now just use my github acount "xwang2713". When the repo is created in "hpcc-systems" we can update this workflow script

Reply:
I kept the account as yours, repo as yours, and then added in the secret token. We should be able to 'test' it on Thursday when we do another set of tags.